### PR TITLE
Fix class name of the read marker on `MessagePanel`

### PR DESCRIPTION
**Pull request by luixxiul**
This PR suggests to fix a read marker's class name, so that it indicates that the marker belongs to `MessagePanel`, not `RoomView`.

It also intends to create `_MessagePanel.pcss` for `MessagePanel.tsx` and hide the zero height element  which just wraps `<hr>` on Percy snapshots.

type: task

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

## Checklist

-   [ ] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: test/components/structures/MessagePanel-test.tsx [hunk 1]
```diff
@@ -353,7 +353,7 @@ describe("MessagePanel", function () {
         const tiles = container.getElementsByClassName("mx_EventTile");
 
         // find the <li> which wraps the read marker
-        const [rm] = container.getElementsByClassName("mx_RoomView_myReadMarker_container");
+        const [rm] = container.getElementsByClassName("mx_MessagePanel_myReadMarker");
```
Comment (luixxiul) [inline test/components/structures/MessagePanel-test.tsx:356]: Having the element named as `mx_MessagePanel_` indeed should look natural.

**Review by andybalaam — APPROVED** [review]
Thank you - this looks good and I will merge it.

Even better would be to split the 3 things you mentioned in the PR description into 3 separate PRs!

**Comment by luixxiul** [comment]
Blocked on https://github.com/matrix-org/matrix-react-sdk/pull/10836

**Comment by luixxiul** [comment]
`permalinks.spec.ts` was updated and it should be fine now.

**Comment by andybalaam** [comment]
Two flakes in Percy caused the failure: https://github.com/vector-im/element-web/issues/25283 and https://github.com/vector-im/element-web/issues/24881

**Comment by andybalaam** [comment]
.. and now we have merge conflicts :-( Should be better when that is fixed, now that https://github.com/matrix-org/matrix-react-sdk/pull/10878 is merged.

**Comment by luixxiul** [comment]
Ideally the merge queue should return an error as soon as a PR is added to the queue, if the conflicts are expected to occur if the PRs on the queue have been successfully merged…

**Comment by luixxiul** [comment]
Here is a link to the run: https://cloud.cypress.io/projects/ppvnzg/runs/13963/overview

**Pull request merged**